.circleci/unittest/linux/scripts/run_test.sh

```diff
@@ -7,4 +7,4 @@ conda activate ./env
 
 export PYTORCH_TEST_WITH_SLOW='1'
 python -m torch.utils.collect_env
-pytest --cov=torchvision --junitxml=test-results/junit.xml -v --durations 20
+pytest -v --durations 20
```

Comment:
we shouldn't remove the `--junitxml=test-results/junit.xml` part, this is what CircleCI uses to provide the "extended" test failure summary https://circleci.com/docs/2.0/collect-test-data/

Reply:
Hmm, I didn't know that. I thought we dump the coverage file as xml. (Something like [this](https://github.com/oke-aditya/quickvision/blob/master/.github/workflows/ci-testing.yml#L41))

I think you are right. Let me rollback this
https://circleci.com/docs/2.0/collect-test-data/#pytest

Reply:
```suggestion
pytest --junitxml=test-results/junit.xml -v --durations 20
```